docs/examples/Electromagnetic Phenomena/helical_motion.py

```diff
@@ -0,0 +1,152 @@
+"""
+======================================================================
+Motion of a charged particle in a combined magnetic and electric field
+======================================================================
+
+A charged particle follows a curved path in a magnetic field.
+In an electric field, the particle tends to accelerate in a direction parallel/
+antiparallel to the electric field depending on the nature of charge on the
+particle.
+In a combined electric and magnetic field, the particle moves along a helical
+path.
+
+In this animation, there's a magnetic and an electric field present in +x
+direction under whose influence the charged particle follows a helical path.
+
+"""
+
+###############################################################################
+# Importing necessary modules
+from fury import window, actor, utils, ui
+import numpy as np
+import itertools
+
+
+###############################################################################
+# Variable(s) and their description-
+# radius_particle: radius of the point that will represent the particle
+#                  (default = 0.08)
+# radius_path: radius of the points that will render the path of the particle
+#                  (default = 0.04)
+# u: initial velocity of the particle
+#    (default = 0.09)
+# a: acceleration of the particle along +x (due to the electric field)
+#    (default = 0.004)
+# t: time (default time i.e. time at beginning of animation  = 0)
+# dt: value by which t is incremented for each call of timer_callback
+#     (default = 0.09)
+# w: angular frequency (default = 0.1)
+# d: phase angle (default = 0.002)
+
+radius_particle = 0.08
+radius_path = 0.04
+u = 0.09
+a = 0.004
+t = 0
+dt = 0.09
+w = 0.1
+d = 0.002
+
+
+###############################################################################
+# Creating a scene object and configuring the camera's position
+scene = window.Scene()
+scene.zoom(1.2)
+scene.set_camera(position=(10, 12.5, 19), focal_point=(3.0, 0.0, 0.0),
+                 view_up=(0.0, 0.0, 0.0))
+showm = window.ShowManager(scene,
+                           size=(800, 600), reset_camera=True,
+                           order_transparent=True)
+showm.initialize()
+
+
+###############################################################################
+# Creating a blue colored arrow to show the direction of propagation of
+# electromagnetic wave
+color_arrow = window.colors.blue  # color of the arrow can be manipulated
+centers = np.array([[0, 0, 0]])
```

Comment:
same as above and below.
I think it should be `charged particle` instead of `EM wave`.

Reply:
Thanks for pointing this out. It's supposed to represent the direction of magnetic and electric field. I rectified it.